From: Peng Li <peng8420.li@gmail.com>
Subject: mm/vmalloc: fix build error
Date: Tue, 18 Nov 2025 23:50:34 +0800

commit 056b93566a35 ("mm/vmalloc: warn only once when vmalloc
detect invalid gfp flags") modify the function call from WARN to WARN_ONCE.

WARN_ONCE input parameter format error, missing ',' tag.
Resulting in the following compilation errors:

mm/vmalloc.c:3940:19: error: expected `)' before `invalid_mask'
 3940 |                   invalid_mask, &invalid_mask, flags, &flags);

After adding the "," tag, the compilation passed.

Link: https://lkml.kernel.org/r/20251118155034.273696-1-peng8420.li@gmail.com
Signed-off-by: Peng Li <peng8420.li@gmail.com>
Cc: David Heidelberg <david@ixit.cz>
Cc: "Uladzislau Rezki (Sony)" <urezki@gmail.com>
Cc: Vishal Moola (Oracle) <vishal.moola@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/vmalloc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/vmalloc.c~mm-vmalloc-warn-only-once-when-vmalloc-detect-invalid-gfp-flags-fix
+++ a/mm/vmalloc.c
@@ -3936,7 +3936,7 @@ static gfp_t vmalloc_fix_flags(gfp_t fla
 	gfp_t invalid_mask = flags & ~GFP_VMALLOC_SUPPORTED;
 
 	flags &= GFP_VMALLOC_SUPPORTED;
-	WARN_ONCE(1, "Unexpected gfp: %#x (%pGg). Fixing up to gfp: %#x (%pGg). Fix your code!\n"
+	WARN_ONCE(1, "Unexpected gfp: %#x (%pGg). Fixing up to gfp: %#x (%pGg). Fix your code!\n",
 		  invalid_mask, &invalid_mask, flags, &flags);
 	return flags;
 }
_
